spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/Scala213BigQueryRDD.java

```diff
@@ -83,6 +97,7 @@ public scala.collection.Iterator<InternalRow> compute(Partition split, TaskConte
             bigQueryClientFactory,
             request,
             options.toReadSessionCreatorConfig().toReadRowsHelperOptions());
+    tracer.startStream();
```

Comment:
Should be in `ReadRowsHelper.readRows()`

Reply:
Fixed